careers/application-process/index.html

```diff
@@ -0,0 +1,292 @@
+                    block__flush-bottom">
+        <h3>Preparing a Résumé to Apply to a Federal Government Position</h3>
+        <p>
+            It is important not to leave something out
+            just because it seems less relevant to you.
+            Federal hiring will always take into account the full breadth of your experience.
+            If something is listed under “minimum qualifications required,”
+            your application must demonstrate that you have
+            experience relevant to that qualification.
+            It is also important that you do not make assumptions about what qualifies.
+            If the announcement says a particular type of experience is required,
+            do not list a similar but different type of experience
+            without explaining how it meets the qualification.
+            If you have that exact experience, always state it.
+        </p>
+        <p>
+            You are welcome to say more than this, of course,
+            and it can help later in the process.
+            Once the process moves past the initial round of review,
+            people will take a closer look at the patterns of your experience
+            rather than just the broadest facts.
+        </p>
+    </section>
+
+    <section class="block
+                    block__padded-top
+                    block__border-top
+                    block__flush-bottom">
+        <h3>Federally Recognized Status and Eligibility for a Position</h3>
+        <p>
+            Sometimes very similar jobs will be posted simultaneously,
+            with one of the primary differences being who is eligible to apply.
+            This difference is based on the applicant’s employment history
+            or other federally recognized status.
+            For example, some jobs are limited to current and former federal employees only.
+            In general, if you see two similar jobs that interest you,
+            we recommend applying through the description
+            that applies most closely to your experience and status.
+            You are, however, welcome to apply to any vacancy at any time.
+        </p>
+    </section>
+
+    <section class="block
+                    block__padded-top
+                    block__border-top
+                    block__flush-bottom">
+        <h3>Federal Ethics Rules and CFPB Ethics Regulations</h3>
+        <p>
+            We hold ourselves to an exemplary standard of integrity in all that we do.
+            Should you join us, you will be subject to federal government-wide ethics rules
+            as well as the additional ethics regulations specific to Bureau employees.
+        </p>
+        <p>
+            CFPB Ethics Regulations will prohibit you and your spouse and minor children
+            from holding financial interests in entities supervised by the CFPB;
+            restrict your ability to work on matters involving certain of your creditors,
```

Comment:
> work on matters involving certain of your creditors

Does this read weirdly to everyone else? Maybe something like

> work on matters involving certain creditors you may have

would be better?